src/Symfony/Component/HttpFoundation/ResponseHeaderBag.php

```diff
@@ -88,12 +88,22 @@ public function replace(array $headers = [])
     /**
      * {@inheritdoc}
      */
-    public function all()
+    public function all(/*string $key = null*/)
     {
         $headers = parent::all();
+
         foreach ($this->getCookies() as $cookie) {
             $headers['set-cookie'][] = (string) $cookie;
         }
+        if (1 === \count(\func_get_args()) && ($key = func_get_arg(0)) && null !== $key) {
```

Comment:
before, we can add a blank line
then: `if (1 <= \func_num_args() && null !== $key = func_get_arg(0)) {`